src/components/node-list/node-list-row.js

```diff
@@ -102,6 +102,8 @@ const NodeListRow = memo(
             'pipeline-nodelist__row--selected': selected,
             'pipeline-nodelist__row--disabled': disabled,
             'pipeline-nodelist__row--unchecked': !checked,
+            'pipeline-nodelist__row--overwrite':
+              active || selected ? false : true,
```

Comment:
Can this be `!(active || selected)`

Reply:
yes it can 👍 I wrote it this way for readability, but on looking back this way is very readable as well - I've updated this